editor/editor_node.cpp

```diff
@@ -6044,9 +6044,14 @@ EditorNode::EditorNode() {
 	EDITOR_DEF("interface/editor/save_on_focus_loss", false);
 	EDITOR_DEF_RST("interface/editor/save_each_scene_on_quit", true);
 	EDITOR_DEF("interface/editor/quit_confirmation", true);
-	EDITOR_DEF("interface/editor/show_update_spinner", false);
 	EDITOR_DEF("interface/editor/update_continuously", false);
+#ifdef ANDROID_ENABLED
```

Comment:
I think we could default it on for the Web editor too:
```suggestion
#if defined(ANDROID_ENABLED) || defined(WEB_ENABLED)
```